tracer/src/Datadog.Tracer.Native/tracer_tokens.cpp

```diff
@@ -227,6 +275,27 @@ void TracerTokens::AddAdditionalLocals(COR_SIGNATURE (&signatureBuffer)[BUFFER_S
     memcpy(&signatureBuffer[signatureOffset], &exTypeRefBuffer, exTypeRefSize);
     signatureOffset += exTypeRefSize;
     signatureSize += 1 + exTypeRefSize;
+
+    if (enable_by_ref_instrumentation)
+    {
+        // Gets the exception type buffer and size
```

Comment:
nit: update the comment

Reply:
done in [1c7c844](https://github.com/DataDog/dd-trace-dotnet/pull/5442/commits/1c7c8440a413483bf13664e87a42bc8669ce4a91)